app/components/WhitelistedDApps/index.js

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+
+import { normalizeUrl } from '../../utils/helpers'
```

Comment:
maybe it would be better use absolute paths? You can configure webpack for just add ```src``` as index, and therefore just need:
```
import { normalizeUrl } from 'utils/helpers' 
```